src/__test__/redux/actions/samples/saveSamples.test.js

```diff
@@ -97,15 +98,24 @@ describe('saveSamples action', () => {
     const store = mockStore(initialState);
     await store.dispatch(saveSamples(mockprojectUuid, newSample));
 
-    const firstAction = store.getActions()[0];
-    expect(firstAction).toMatchSnapshot();
+    const actions = store.getActions();
+
+    // First action sets up saving status
+    expect(actions[0].type).toBe(SAMPLES_SAVING);
+
+    // Second state generates error
+    expect(actions[1].type).toBe(SAMPLES_ERROR);
+
+    // Thirdd state emits notification
+    expect(actions[2].type).toBe(NOTIFICATIONS_PUSH_MESSAGE);
   });
 
   it('Dispatches project guards correctly', async () => {
```

Comment:
rename to samples, not project and use a different word instead of `guards`

Reply:
changed to `pre and post actions`